### ISSUE TITLE
airports

### ISSUE THREAD

**Issue by Amateur-God** [jetelain/ArmaRealMap#259]
Runways are just grass, they  aren't getting any actual airport stuff like runways or ATC, terminals, hangars etc

**Comment by WirtsLegs** [comment]
Yeah Ive been finding I get a sorta gravel runway and then usually just different coloured grass for the taxiways etc

would be great if they could be identified and done properly
